aips/aip-50.md

```diff
@@ -0,0 +1,54 @@
+---
+aip: 50
+title: Allow direct commission change vesting contract
+author: junkil-park, movekevin, wintertoro
+discussions-to: https://github.com/aptos-foundation/AIPs/issues/249
+Status: In Review
+type: Standard Framework
+created: 09/28/2023
+---
+
+# AIP-50 - Change commission rates in delegation pools
+
+## Summary
+Currently the commission rate is set by the pool owner at initiation of pool. We propose allowing the pool owner to change the commission rate after delegation pool creation.
+
+### Goals
+This feature will increase flexibility of pool owners over the network / commission rates. Pool owner can adjust operator commission rates as appropriate, based on market conditions and commercial discussions. As such, this will facilitate staking pools to participate in market dynamics. I.e. the pool can be set at a lower commission to attract more stakers or raise commission when needed. Operators can call the function themselves if they are also the pool owner, or if not, work with the pool owner they are operating for.
+
+### Out of Scope
+Operators will not be able to change commission rate. In our existing design setup, Operators are designated by pool owners.
+
+### Impact
+Delegation pool owners and corresponding operators.
```

Comment:
we should callout the impact on delegator too, people who stake to the delegation pool previously would assume the rate is fixed and not changeable, but now they need to watch over the commission rate once for a while to make sure the operator did not overcharge them.

Reply:
That's a good point! I added that.